README.md

```diff
@@ -0,0 +1,136 @@
+## Empathy
+
+![image](https://user-images.githubusercontent.com/240083/37926477-4c464bde-30ec-11e8-8a0f-1d2b4b7521d0.png)
+Empathy is a minimalist tool that generates a folder of "asset" dependencies for
+an NPM package. It is designed in the spirit of the 
+[`npm/npm#npm-asset`](https://github.com/npm/npm/tree/npm-asset) work being done
+by NPM.
+
+Asset dependencies are "Web Ready" NPM packages whose specifiers have been
+adjusted to point to their dependencies at the appropriate paths in the local
+file tree.
+
+Empathy can also transform local source files to use the correct "name" 
+specifiers when it comes time to publish those sources to NPM.
+
+### Usage
+
+Add `empathy` as a dev dependency of your package:
+
+```
+$ npm i --save-dev @0xcda7a/empathy
+```
+
+Then include your assets as dependencies in your `package.json`. For example, 
+this partial manifest includes an asset dependency on Polymer v3.0.0-pre:
+
+```js
+{
+  "name": "cool-dogs-package",
+  "version": "1.0.0",
+  // ...
+  "dependencies": {
+    "@polymer/polymer": "v3.0.0-pre.12",
+    // and other dependencies...
+  },
+  // ...
+}
+```
+
+#### Installing assets from NPM
+
+The easiest way to install assets from NPM with `empathy` is to run the 
+`install` command:
+
+```
+$ empathy install
+```
+
+You should see console output that shows progress generating your asset 
+dependencies. When it is done, you will have an `assets` folder in your current
+working directory. This folder contains all of your asset dependencies, which
+should be importable by path specifiers.
+
+You can also specify the output folder where assets should be generated to:
+
+```
+$ empathy install -a my/asset/dir
+```
+
+You can restrict which packages are installed as assets. To select specific
+packages to convert, use `--only`:
+
+```
+$ empathy install --only \@polymer/polymer
+```
+
+You can also select specific packages to ignore using `--ignore`:
+
+```
+$ empathy install --ignore express
+```
+
+For extreme mindfulness, add empathy to your `postinstall` step:

```

Comment:
* What's the problem with asset-izing sources which are unused? Besides it being slow etc, are there other side effects?
 * This seems brittle. For example, it seems useful to put `empathy install` in the `build` script of your application. In this case, these ignore/onlys won't be 'saved' in any kinda way.